src/stories/pill.stories.mdx

```diff
@@ -0,0 +1,54 @@
+import { Canvas, Meta, Story } from '@storybook/addon-docs'
+
+<Meta
+  title='Elements/Pill'
```

Comment:
You can get the remove event to show up in the interactions tab in the canvas view by adding this to Meta

```
  parameters={{
    actions: {
      handles: ['remove'],
    },
  }}
  ```
  
  Useful for users who are trying to see how the details will be surfaced to them